test/uk/gov/hmrc/helptosavereminder/controllers/ReminderControllerSpec.scala

```diff
@@ -60,16 +57,14 @@ class HtsUserUpdateControllerSpec extends AuthSupport with TestSupport {
       "org.apache.logging" -> "ERROR",
       "com.codahale"       -> "ERROR")
 
-  private val bindModules: Seq[GuiceableModule] = Seq(new PlayModule)
-
   val mockRepository = mock[HtsReminderRepository]
 
   val mcc: ControllerComponents = fakeApplication.injector.instanceOf[ControllerComponents]
 
-  def mockSendAuditEvent(event: HTSEvent, nino: String) =
+  def mockSendAuditEvent(event: HTSEvent) =
```

Comment:
Was a nino supposed to being sent in the audit event? It wasn't being used so it was making a compiler warning.